src/quo/components/internal_link_card/community/view.cljs

```diff
@@ -0,0 +1,101 @@
+(ns quo.components.internal-link-card.community.view
```

Comment:
this name space is wrong, we have a guideline to keep the mapping 1-1 with figma. Please move this to `quo.components.links.internal-link-card.community.view`